### DISCUSSION TITLE
MSTN connector when importing ORD files

### DISCUSSION THREAD

**Discussion by jecynn** [iTwin/community#11] [Q&A] [answered]
Hi,

I just created a test iTwin/iModel and imported an ORD project into the dev.bentley.com environment for testing purposes,

This is what I get in the File Synchronization tab
![image](https://github.com/iTwin/community/assets/58005663/2520913c-39b6-48b7-b2a4-3b1919d82db6)

Shouldn't these files be processed by the CIVIL connector instead of the MSTN connector ?
Should it be cause for concern ?

This is from the Bentley Synchronization API doc
![image](https://github.com/iTwin/community/assets/58005663/3ebd6d5c-a20c-4308-b52f-10f5fc11ec1b)

**Comment by williamkbentley** [comment, marked as answer]
Hello,

The File Synchronization in My iTwins is a fairly simplified implementation of they Synchronization API. It uses basic mappings of *.dgn -> MSTN connector. So from that standpoint, this is working as designed. 

In the future we may add functionality to allow the user to select the connector type when uploading, instead of these broad defaults.
